scripts/build_package.py

```diff
@@ -700,6 +700,11 @@ def build_package(repository_root, build_package_config, env=None):
     target = os.path.join(package_root, package_layout['www'])
     copy_tree(source, target)
 
+    # Run doxygen.
```

Comment:
Why do we call doxygen here? It is called in the Makefile https://github.com/Ericsson/codechecker/blob/de1ab87495745dac77065dd34083c668e392ee39/Makefile#L20